### DISCUSSION TITLE
課題05についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#11] [Q&A]
サンプルの真似をして作成してみたのですが、正確に合っているのかわかりません。

はじめ、10行目は
const practice05Map = new Map<string, number>()となっていたのですが、
11行目以降でエラーがでていたため、
const practice05Map = new Map<string, string>()のように、「number」→「string」へ変更いたしました。

**Comment by nsc-kiwamu** [comment]
以下で回答済
https://github.com/nsc-kiwamu/traning-takeda/pull/10#issuecomment-1175161849

**Reply by TakedaAyano** [reply]
ご回答ありがとうございます。

承知いたしました。numberを指定し、''を外して再度実行してみます。
